.github/workflows/npm-download-metrics.yml

```diff
@@ -0,0 +1,27 @@
+name: NPM Download Metrics
+
+on:
+  schedule:
+    - cron: "30 3 * * *" #runs at 03:30 UTC or 09:00 IST everyday
+  workflow_dispatch: 
+ 
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout cli repo
+        uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8 
+      - name: Execute py script 
+        env:
+          DD_SITE: datadoghq.com
+          DD_API_KEY: ${{ secrets.DATADOG_API_KEY }}   
+        run: |
+          pip3 install datadog-api-client
+          pip3 install requests
+          pip3 install datetime
+          python3 .github/scripts/npm_metrics.py
```

Comment:
Can we club all metric related files to a common directory structure:
```
.github/scripts/metrics/[]
.github/workflows/metrics/[] // If github can parse from a subdir
```

Reply:
Sure